### PR TITLE
Update extconf.rb to check for termux

### PR DESCRIPTION
**Pull request by Auxilus**

### CODE REVIEW

File: src/ruby/ext/grpc/extconf.rb [hunk 1]
```diff
@@ -62,20 +64,32 @@
 ENV['ARCH_FLAGS'] = '-arch i386 -arch x86_64' if RUBY_PLATFORM =~ /darwin/
 ENV['CFLAGS'] = '-DGPR_BACKWARDS_COMPATIBILITY_MODE'
 
-output_dir = File.expand_path(RbConfig::CONFIG['topdir'])
-grpc_lib_dir = File.join(output_dir, 'libs', grpc_config)
-ENV['BUILDDIR'] = output_dir
+if ENV['PREFIX'] == '/data/data/com.termux/files/usr'
```
Comment (timwr) [inline src/ruby/ext/grpc/extconf.rb]: Why not just `if ENV['PREFIX']`? no need to check the path

Comment (Auxilus) [reply src/ruby/ext/grpc/extconf.rb]: Yeah but linux has different $PREFIX than termux, so just to be accurate...

BTW this one should work right?

**Comment by grpc-testing** [comment]
Thanks for your pull request. The automated tests will run as soon as one of the admins verifies this change is ok for us to run on our infrastructure.

File: src/ruby/ext/grpc/extconf.rb (hunk 1 above)
Comment (timwr) [reply src/ruby/ext/grpc/extconf.rb]: are you sure Linux has a $PREFIX?
You could also check $TERMUX_PREFIX

**Comment by grpc-testing** [comment]
Thanks for your pull request. The automated tests will run as soon as one of the admins verifies this change is ok for us to run on our infrastructure.

File: src/ruby/ext/grpc/extconf.rb (hunk 1 above)
Comment (Auxilus) [reply src/ruby/ext/grpc/extconf.rb]: Oh i see..i guess linux has $PATH not $PREFIX

**Comment by timwr** [comment]
Many thanks @Auxilus !

**Comment by Auxilus** [comment]
👍

**Comment by Auxilus** [comment]
Just tested the PR on termux, its working..

**Comment by nathanielmanistaatgoogle** [comment]
Seven commits to change three lines in one file? Please squash commits?

**Comment by Auxilus** [comment]
I don't know how to squash commits ( im not on terminal btw ) should I just open new PR with only the last commit?

**Comment by Auxilus** [comment]
I see that only people with write access to repository can squash the commits, so if you can then please go ahead, thanks

**Comment by Op3n4M3** [comment]
@Auxilus, squashing commits can be done on your own branch in your forked repo.  Since this PR is still open you can squash your commits with an interactive rebase on your branch and squash to only the first commit.  Then after verification of resulting diff a force push to your original branch will update this PR.

https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History

```
git checkout patch-2
git rebase -i master
```

**Comment by shai-dinosaur-guy** [comment]
yee

**Pull request closed without merging**